doc: document generate with working directory. (#41)
terramate-io/terramate-docs: cli/reference/cmdline/generate.md

## Examples

> Generate files for all configurations defined in the current directory and all of its subdirectories.

Reviewer: ```suggestion
Generate files for all configurations defined in the current directory and its subdirectories.
```

```bash
terramate generate
```

> To generate Terraform configurations only for configirations defined within a specific subdirectory, use the `-C` (or `--chdir`) flag followed by the desired directory path.

Reviewer: ```suggestion
To generate Terraform files targeting configurations defined within a specific subdirectory, use the `-C` (or `--chdir`) flag followed by the desired directory path.
```

```bash
terramate -C some/dir generate
```

By default, Terramate returns status code = 0 when it succeeds at ensuring that all files are updated.

> If you wish to know when files were touched in disk, then use the `--detailed-exit-code` flag.

Reviewer: ```suggestion
Use the `--detailed-exit-code` flag to check when files were modified on disk.
```
